### PR TITLE
Feature about page

### PR DESCRIPTION
**Pull request by KoffiKC**
I made the 'HotSoup' title on the homepage a pressable that sends you to the about page, as this is in line with the figma. Though it was mentioned during a meeting that having the pressable there might not be the best. I imagine once tab navigation is implemented we can remove it but let me know if I should change it before merging.

### CODE REVIEW

File: src/screens/Home.tsx [hunk 1]
```diff
@@ -16,7 +16,10 @@ export default function Home() {
 
     return (
         <View style={styles.container}>
+            <Pressable
+                onPress={() => navigation.navigate('AboutPage')}>
             <Text style={styles.title}>HotSoup</Text>
```
Comment (lindentree) [inline src/screens/Home.tsx:21]: Please indent this line properly: `<Text` style={styles.title}>HotSoup</Text>`

@rweitzman I think this code can be merged into dev as is, and Jaideep can merge it to his. However, we might want to review the design decision to make the title a navigable button.

File: src/screens/AboutPage.tsx [hunk 2]
```diff
@@ -0,0 +1,64 @@
+import { StatusBar } from 'expo-status-bar';
```
Comment (lindentree) [inline src/screens/AboutPage.tsx:1]: This file should probably be named just About.tsx

Comment (KoffiKC) [reply src/screens/AboutPage.tsx:1]: I've updated this but wont do another push until tomorrow after getting other input.

Comment (KoffiKC) [reply src/screens/AboutPage.tsx]: I updated it again to Info instead of about. Thank goodness there's a search feature lol.

**Review by rweitzman — APPROVED** [review]

**Pull request merged**